include/swift/AST/ASTScope.h

```diff
@@ -618,7 +610,6 @@ class Portion {
   getChildlessSourceRangeOf(const GenericTypeOrExtensionScope *scope,
                             bool omitAssertions) const = 0;
 
-  /// Returns isDone and isCascadingUse
```

Comment:
What does it return now? Just isDone? Unless a function name gives the predicate, I think a comment can really help.

Reply:
I believe it returns `isDone` given that it calls `lookupMembersOf`, and that function no longer computes the cascading bit. I'll add this back once the testing infrastructure gives me the green light.

Reply:
Thanks!